modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderContext.tsx

```diff
@@ -29,6 +29,20 @@ interface ObjectFolderContextProviderProps
 const ObjectFolderContext = createContext({} as ObjectFolderContextProps);
 
 const initialState = {
+	changeModalVisibility: {
+		addObjectDefinition: false,
+		addObjectField: false,
+		addObjectFolder: false,
+		addObjectRelationship: false,
+		deleteObjectDefinition: false,
+		deleteObjectFolder: false,
+		deleteObjectRelationship: false,
+		editObjectDefinitionExternalReferenceCode: false,
+		editObjectFolder: false,
+		moveObjectDefinition: false,
+		publishObjectDefinitions: false,
+		redirectToEditObjectDefinitionDetails: false,
+	},
```

Comment:
Hey @Artartur, could you please rename `changeModalVisibility` to `modelBuilderModals` and use the interface `ModelBuilderModals` instead declare all the properties? So:

`modelBuilderModals: {} as ModelBuilderModals`